examples/kokkos-benchmarks/standalone/notype/gups.py

```diff
@@ -0,0 +1,86 @@
+import argparse
+import random
```

Comment:
same comment as earlier: keep new examples for the next PR, so we can review them independently

Reply:
These were no-types versions of existing examples. As discussed over slack I am replaced the original examples themselves now. Resolving this.